### DISCUSSION TITLE
Devices which can boot into koreader

### DISCUSSION THREAD

**Discussion by avsej** [koreader/koreader#7449] [Q&A] [answered]
I'm wondering if there any devices like BQ Cervantes where I can skip stock frontend completely and boot koreader as if it was default application.

I'm asking because it seems like the device is not available anymore.

**Comment by pazos** [comment, marked as answer]
any kobo with an internal sdcard and some hacking. Keep in mind that storage is much slower than the internal emmc the Cervantes 4 has.

But yeah, a kobo. If you don't care about the stock os you can skip the 3rd partition entirely (formated as fat32) and even the 2nd partition (recovery) and boot yourself a minimal rootfs based on busybox. 

I did that plenty of times for a ngo. Pretty much generic [Buildroot](https://buildroot.org/) with a few udev patches. If you don't care about hotplug events you can use mdev instead (part of busybox).

You need a few bind mounts to simulate the same paths used by stock os because KOReader on kobos relies on their availability.

The Clara HD fullfits your needs but isn't worth the update from a Cervantes 4.
